File: helper/win32/KeeWebHelper/InputCommands/UnknownCommand.cs
using System;

namespace KeeWebHelper.InputCommands
{
    class UnknownCommand : InputCommandBase
    {
        public string Name { get; set; }

        public UnknownCommand(string name)
        {
            Name = name;
        }

        public override void Execute()
        {
            Console.Error.WriteLine("Unknown command: {0}", Name);
        }
    }
}

File: helper/darwin/src/KeeWebHelper/InputCommands/UnknownCommand.m
#import "UnknownCommand.h"

@implementation UnknownCommand

- (id)initWithName:(NSString *)aName {
    self = [super init];
    if (self) {
        name = aName;
    }
    return self;
}

- (void)execute {
    NSLog(@"Unknown command: %@", name);
}

@end
